java/org/apache/catalina/session/FileStore.java

```diff
@@ -136,8 +136,8 @@ public int getSize() throws IOException {
         // Figure out which files are sessions
         int keycount = 0;
         if (files != null) {
-            for (int i = 0; i < files.length; i++) {
-                if (files[i].endsWith(FILE_EXT)) {
+            for (String file1 : files) {
+                if (file1.endsWith (FILE_EXT)) {
                     keycount++;
```

Comment:
`file1` looks odd here. I'd change the `file` above to `dir` and use `file` here for consistency with other for loops.

Reply:
Renamed `files[]` source as `dir` Please let me know if you think otherwise i.e. to change files as dir.